hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotDiffManager.java

```diff
@@ -1194,23 +1241,28 @@ private boolean areKeysEqual(WithObjectID oldKey, WithObjectID newKey) {
   }
 
   /**
-   * check if the given key is in the bucket specified by tablePrefix map.
+   * Get table prefix given a tableName.
    */
-  boolean isKeyInBucket(String key, Map<String, String> tablePrefixes,
-                        String tableName) {
+  private String getTablePrefix(Map<String, String> tablePrefixes,
+                                 String tableName) {
```

Comment:
Alignment is bit off.
```suggestion
  private String getTablePrefix(Map<String, String> tablePrefixes,
                                String tableName) {
```

Reply:
done